.azure-pipelines/client.yml

```diff
@@ -2,32 +2,27 @@
 #  'build_targeting_string' comma separated glob strings selecting the packages that should be built. EG: "azure-keyvault,azure-mgmt-batch" or "azure-*"
 #  'codecov-python-repository-token' Is the codecov.io repository token.
 #  'long_running_tests' Is a boolean indicating that the long-running tests should be executed. 
+#  'EnableDetectorPip': true . Required for Component Governance Support
 
 trigger:
 - master
+variables:
+  python.version: '3.6'
```

Comment:
Per the [guidelines](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki?wikiVersion=GBwikiMaster&pagePath=%2FAzure%20DevOps%20Pipeline%20Guidance&pageId=2&anchor=variables), we decided to standardize on `CamelCase` for variables.  So please rename this from `python.version` to `PythonVersion`.  Existing pipelines will eventually be updated to match this standard.